plaidml2/bridge/keras/__init__.py

```diff
@@ -324,7 +324,44 @@ def backend():
 
 @_log_call
 def batch_dot(x, y, axes=None, name=None):
-    _report_unimplemented('batch_dot')
+    X = x.tensor
+    Y = y.tensor
+    if isinstance(axes, int):
```

Comment:
Could we use `six.integer_types` here to make it work on python2 + python3?

Reply:
Certainly